### DISCUSSION TITLE
How to keep the # before headings or make them more distinguishable.

### DISCUSSION THREAD

**Discussion by Nriver** [TriliumNext/Trilium#3490] [Q&A] [answered]
I noticed in the lastest 0.58.3-beta, the # before headings are not shown anymore. It's quite convenient to have them as a mark that I can tell which level the heading is just by a glance. Is there a way go get it back? Or some other way to make different levels of headings more distinguishable.

**Comment by sigaloid** [comment, marked as answer]
Did your header style get changed? Check options -> text notes -> heading style and ensure it's still "markdown-style".

**Reply by Nriver** [reply]
Thank you!
